config/secrets.yml

```diff
@@ -16,6 +16,16 @@ default: &default
   aws_access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
   aws_secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
   aws_bucket: <%= ENV['AWS_BUCKET'] %>
+  enable_persona: <%= ENV['ENABLE_PERSONA'] || true %>
```

Comment:
Where else would we put them?

On Wed, 20 May 2015 at 13:11 James Kiesel <notifications@github.com> wrote:

> In config/secrets.yml
> <https://github.com/loomio/loomio/pull/2172#discussion_r30664139>:
>
> > @@ -16,6 +16,16 @@ default: &default
> >    aws_access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
> >    aws_secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
> >    aws_bucket: <%= ENV['AWS_BUCKET'] %>
> > +  enable_persona: <%= ENV['ENABLE_PERSONA'] || true %>
>
> Don't think we want the secrets file to have opinions about defaults for
> the ENV values, unless I'm missing something here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2172/files#r30664139>.
>